### PR TITLE
update download tasks

### PR DESCRIPTION
**Pull request by rekup**
- various tools are now installed beforehand, no need to download & extract anymore

### CODE REVIEW

File: content/en/docs/01/labs/11.md [hunk 1]
```diff
@@ -6,42 +6,22 @@ sectionnumber: 1
 
 In this first lab you are going to download the `node_exporter` and configure Prometheus to scrape it.
 
+* Configure Prometheus to scrape the metrics of the [`node_exporter`](https://github.com/prometheus/node_exporter) installed on your machine. The `node_exporter` metrics are available at `http://localhost:9100/metrics`
```
Comment (bliemli) [inline content/en/docs/01/labs/11.md:9]: Why move the task description outside of the task?

File: content/en/docs/03/_index.md [hunk 2]
```diff
@@ -8,50 +8,30 @@ sectionnumber: 1
...
+# reload
+systemctl reload alertmanager.service
+```
 
+The configuration file of alertmanager is located here: `/etc/alertmanager/alertmanager.yml`
 
 ### Configuration
 
-The configuration of Alertmanager is done using a YAML config file and CLI flags. The Alertmanager tarball we downloaded earlier includes a very basic configuration file:
+The configuration of Alertmanager is done using a YAML config file and CLI flags. Take a look at the very basic configuration file at `/etc/alertmanager/alertmanager.yml`:
```
Comment (bliemli) [inline content/en/docs/03/_index.md:34]:
```suggestion
Alertmanager's configuration is done using a YAML config file and CLI flags. Take a look at the very basic configuration file at `/etc/alertmanager/alertmanager.yml`:
```

**Review by bliemli — COMMENTED** [review]
Two more suggestions, everything else looks very good :+1:

**Comment by rekup** [comment]
Thanks for the review

**Review by bliemli — APPROVED** [review]

**Pull request merged**